Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -331,8 +331,13 @@ Page* WebChromeClient::createWindow(Frame& frame, const WindowFeatures& windowFe
     };
 
     WebFrame* webFrame = WebFrame::fromCoreFrame(frame);
+    
+    std::optional<WebCore::GlobalWindowIdentifier> openerWindowID;
+    if (auto *window = frame.window()) {
+        openerWindowID = window->identifier();
```

Comment:
I don't think this is correct in the case where "noopener" is specified. Should we really always be passing an openerWindowID even when `noopener` was specified and `popup.opener` will be null?